gen-imprecise.js

```diff
@@ -0,0 +1,23 @@
+var plural = require('plural')
+
+var unprecise_units = [
+  "bag", "bar", "bottle", "bowl", "breast", "bulb", "bun", "bunch", "can",
+  "carton", "cone", "clove", "container", "large", "medium", "mini", "small",
+  "cube", "fillet", "head", "jar", "package", "packet", "pack", "patty",
+  "piece", "portion", "servings", "roll", "slice", "handful", "pinch",
+  "touch","slice", "envelope", "sprig", "sheet"
+];
+
+var l = unprecise_units.concat(['dash','splash','unit_']).join(' / ') 
```

Comment:
* Couldn't we just add dash, splash & unit_ into the `unprecise_units` array above, instead of concating.
* Why does unit_ have a underscore?